addons/account/models/ir_attachment.py

```diff
@@ -175,3 +175,10 @@ def action_download_xsd_files(self):
         # To be extended by localisations, where they can download their necessary XSD files
         # Note: they should always return super().action_download_xsd_files()
         return
+
+    def _post_add_create(self, **kwargs):
+        move_id = self.res_id
+        if self.res_model == 'account.move':
+            move = self.env['account.move'].search([('id', '=', move_id)], limit=1)
+            move._check_and_decode_attachment(self)
```

Comment:
```suggestion
        if self.res_model == 'account.move':
            self.env['account.move'].browse(self.res_id)._check_and_decode_attachment(self)
```

I saw that in mail module, they are handling this method as a possible multi (as we don't have a `self.ensure_one()`). Even if it seems that we don't use this function with several record in the same recordset, it could be interesting to do something more robust?

https://github.com/odoo/odoo/blob/ed4e8f5011dd4534d20841252fc52d6a07b5ad35/addons/mail/models/ir_attachment.py#L36-L41

```suggestion
        move_attachments = self.filtered(lambda attachment: attachment.res_model == 'account.move')
        moves_per_id = self.env['account.move'].browse([record.res_id for record in move_attachments]).grouped('id')
        for record in move_attachments:
            moves_per_id[self.res_id]._check_and_decode_attachment(record)
```
(you have the idea, I'll let you fine tune it :smile:)